src/tlo/methods/skeleton.py

```diff
@@ -148,6 +149,16 @@ def report_daly_values(self):
         """
         raise NotImplementedError
 
+    def report_prevalence(self):
```

Comment:
btw - its' great you've put this here --- but this is not the base class of Module. That' found in core.py. That's where we can define the 'default' forms of the functions for the disease module. So I think for report_prevalence we can have a function there which just return None. That way, if a disease module doesn't define its own version of that function (over-writing the default) there is still something to call. We just need to add some logic so make sure that when we get a None returned, we know what it means and handle accordingly.

Reply:
Ah I see - I got confused, because when I looked at the one in core.py, there was no report_dalys function, so I supposed that I misinterpreted what you said (as it was to be like the report_daly function). I could add report_dalys whilst I'm at it, too, then?